specification/semantic_conventions.md

```diff
@@ -99,7 +99,16 @@ For each `InstrumentationLibraryLogs` instance:
 - `name` - MUST be set to `otel.profiling`
 - `version` - MUST be set to `0.1.0`
 
-The resource field in `ResourceLogs` MUST contain the [`telemetry.sdk.language`](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk) attribute.
+The resource field in `ResourceLogs`:
+
+- MUST contain following attributes:
+  - [`telemetry.sdk.language`](https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/resource/semantic_conventions#telemetry-sdk)
```

Comment:
this is already required via https://github.com/lachmatt/gdi-specification/blob/clarify-resource-attribute-requirements/specification/semantic_conventions.md#opentelemetry-resource-attributes

Reply:
It was already there, removed in https://github.com/signalfx/gdi-specification/pull/206/commits/dba1bff7c947c20ad56fab6607df000f370fd80c